src/views/RecIM/views/Activity/index.jsx

```diff
@@ -444,6 +452,15 @@ const Activity = () => {
               individualSport
               activityID={activityID}
             />
+            <WaiverForm
+              username={profile.AD_Username}
+              createSnackbar={createSnackbar}
+              onClose={() => {
+                setReload((prev) => !prev);
+              }}
+              openWaiverForm={openWaiver}
+              setOpenWaiverForm={(bool) => setOpenWaiver(bool)}
```

Comment:
```suggestion
              setOpenWaiverForm={(w) => setOpenWaiver(w)}
```
Naming variables after a type is not the best convention because it quickly confuses the reader. My suggestion is to use a one-character, throw-away, variable name.